modules/matterhorn-event-comment/src/main/java/org/opencastproject/event/comment/persistence/EventCommentDatabaseServiceImpl.java

```diff
@@ -381,28 +410,34 @@ public void repopulate(final String indexName) throws Exception {
       current[0] = 1;
       logger.info("Re-populating index '{}' with comments for events. There are {} events with comments to add",
               indexName, total);
-      final int responseInterval = (total < 100) ? 1 : (total / 100);
```

Comment:
AFAIK, that mechanism was introduced by UCT by intention and should prevent a very large number of ActiveMQ messages being sent.

@smarquard: Am I correctly remembering this? Can it be removed?